packages/components/components/elvis-timepicker/src/react/elvia-timepicker.tsx

```diff
@@ -51,6 +52,19 @@ export const Timepicker: React.FC<Partial<TimepickerProps>> = ({
   inlineStyle,
   webcomponent,
 }) => {
+  const lang = useLanguage();
+
+  const labels =
+    lang === 'no'
+      ? {
+          label: label ?? 'Velg tid',
+          timeSelector: 'Tidvelger',
+        }
+      : {
+          label: label ?? 'Select time',
+          timeSelector: 'Time selector',
```

Comment:
I suggest setting this to "Time picker" to match the component name 😛
```suggestion
          timeSelector: 'Time picker',
```